src/forms/TopComponent/TopComponent.styled.ts

```diff
@@ -43,7 +43,7 @@ export const CheckboxWrapper = styled.div`
 
 export const StyledColumn = styled(Column)`
   background-color: white;
-  box-shadow: 0px 1px 10px #0000001a;
+  border: 2px solid ${theme.colors.gray200};

```

Comment:
The ticket mentions that the border should only be on the bottom, not on top or sides. Please let's make this change.